packages/@aws-cdk/aws-codepipeline-actions/lib/ecr/source-action.ts

```diff
@@ -104,7 +110,7 @@ export class EcrSourceAction extends Action {
     return {
       configuration: {
         RepositoryName: this.props.repository.repositoryName,
-        ImageTag: this.props.imageTag,
+        ImageTag: imageTag,
```

Comment:
This line changes existing behaviour as 'latest' is now returned if no imageTag is provided. This may or may not be desired? Please check

Reply:
Yes, this is not desirable. Let's leave this line as-is.

Reply:
Actually, I just realized, we can't leave this line-as is, because it will most likely fail for `''`:

```suggestion
        ImageTag: this.props.imageTag ? this.props.imageTag : undefined, // `''` is falsy in JS/TS
```